javascript/eslint-config-node/index.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  env: {
+    node: true,
+    es6: true,
+  },
+  extends: [
+    "plugin:prettier/recommended",
+    "eslint:recommended",
+    "plugin:node/recommended",
+  ],
+  plugins: ["prettier"],
+  rules: {
+    quotes: [
+      "error",
+      "single",
+      {
+        avoidEscape: true,
+      },
+    ],
+    "comma-dangle": [
+      "error",
+      {
+        arrays: "always-multiline",
+        objects: "always-multiline",
+        imports: "always-multiline",
+        exports: "always-multiline",
+        functions: "never",
+      },
+    ],
+    "object-curly-newline": "off",
+    "no-confusing-arrow": "off",
+    "arrow-parens": "off",
+    "arrow-body-style": ["error", "as-needed"],
+    "function-paren-newline": "off",
+    indent: "off",
+    "no-mixed-operators": "off",
+    "no-plusplus": "off",
+    "prettier/prettier": "error",
```

Comment:
ja osobne celkom preferujem rozlisovat lint errors a warnings. prettier ako formatting je warning, error su vazne chyby ako nedeklarovana premena.... tak ked pisem kod, tak vidim hned, ze tymto sa mam zaoberat hned (errors) a warningy ignorujem, lebo to doriesi prettier pri save.

ale samozrejme na urovni lintera pouzivat --max-warnings 0